### ISSUE TITLE
Option for KeePassXC to detach from the command line

### ISSUE THREAD

**Issue by SirAthos** [keepassxreboot/keepassxc#9231]
## Summary
There is no good way to start KeePassXC.exe from a Windows .bat script and supply a password at the same time

## Detail
When starting KeePassXC, you can pipe a password to it and have it open a database. However, after opening the database, the process remains in the foreground so the script hangs forever (i.e. until KeePassXC exits).

There is a way to launch a process in the background via the `start` command. However, when using that, KeePassXC is unable to read the password from stdin anymore, as it is detached too early by the interpreter.

Ideally, KeePassXC would provide a command line argument `--background` which enables it to self detach from the shell after processing every other argument, reading password from stdin, opening the database etc.

**Comment by droidmonkey** [comment]
`start keepassxc `

Then

`echo password|keepassxc ....`

Basically start the program, then do a second call with the pipe password which will send it to the running instance and then exit.

**Comment by SirAthos** [comment]
I already tried that. It doesn't send the password over to the other instance. Database name and keyfile info are sent correctly.
This would be an acceptable solution to the issue though.

**Comment by droidmonkey** [comment]
Are you sure you are doing it correctly? You need to make sure your command looks like:

`echo password| keepassxc --pw-stdin ....`

Note the lack of a space and no quotes on the echo command and pipe. 

You also cannot use UTF-8 passwords from the windows command line without some tweaks.

**Comment by SirAthos** [comment]
I'm using the same command that works when there is no other KeePassXC instance (but doesn't return until I quit it), so the password isn't the problem. When another instance is already started and I run this, the older instance pops up with the "open database" dialog, and prefills the fields for database and keyfile, but the password field is empty.
(If the password was wrong, that field would not be empty, and an error message saying "wrong password" or something to that effect would appear above in red. I've run into that too while trying a million different variations.)

**Comment by droidmonkey** [comment]
I agree the field would at least be filled. If you run the command with the password stdin outside the batch file does it work? Might have a bug here...

**Comment by SirAthos** [comment]
If I try outside of a batch file I'll have to edit the escape characters. (After this weekend, I'm pretty sure in hell people will be forced to write complex .bat files. But I digress.)

What I do have is a .bat file that I can run unmodified and:
* it launches KeePassXC and opens the database correctly if there is no other instance running - but doesn't return
* it runs a second KeePassXC which returns immediately, but doesn't open the database correctly

```
setlocal enableDelayedExpansion
set "PW=My password with escaped special characters"
cmd /v:on /c "echo ^!PW^!"| "C:\Program Files\KeePassXC\KeePassXC.exe" --pw-stdin --keyfile X:\Path\To\Keyfile Y:\Path\To\Database.kdbx
```

Edit: I went through re-escaping so I could run this outside of the batch file. Same result as above.

**Comment by droidmonkey** [comment]
OK I see the problem, we never actually send the key file or the password to the running instance. It is not even a feature we support. The key file populates because it is remembered, most likely. I was mistaken, I thought we supported this feature, my apologies.

**Comment by SirAthos** [comment]
No worries. Now we can have two feature requests instead of one. :)

When you say "not supported", do you mean it isn't implemented at this time, or it was decided never to be implemented?

**Comment by droidmonkey** [comment]
Just not implemented
